docs/data-sources/gmail.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: '📬 GMAIL'
+---
+
+To use GMAILLoader you must install the extra dependencies with `pip install --upgrade embedchain[dataloaders]`.
```

Comment:
Can we please do `GmailLoader` everywhere instead of `GMAILLoader` ?

Reply:
Cool